po/extra/it.po

```diff
@@ -92,11 +92,11 @@ msgstr ""
 
 #: data/com.github.elfenware.obliviate.appdata.xml.in:24
 msgid "First AppCenter Release 🎉️"
-msgstr ""
+msgstr "Prima versione AppCenter 🎉️"
 
 #: data/com.github.elfenware.obliviate.appdata.xml.in:26
 msgid "Add: Clear clipboard 30 seconds after copying password"
-msgstr ""
+msgstr "Aggiunto:Cancella gli appunti 30 secondi dopo aver copiato la password"
```

Comment:
Should we put a space after the colon here? What is the convention in Italian?

Reply:
Hi,
Fixed!